### PR TITLE
Update storage account help

### PR DESCRIPTION
**Pull request by troydai**
Partially react to https://github.com/Azure/azure-cli/issues/4347

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [ ] Each command and parameter has a meaningful description.
- [ ] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by troydai** [comment]
@tjprescott can you help me with the words?

File: src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_command_type.py [hunk 1]
```diff
@@ -24,9 +24,10 @@ def cli_storage_data_plane_command(name, operation, client_factory, transform=No
     group_name = 'Storage Account'
     command.add_argument('account_name', '--account-name', required=False, default=None,
                          arg_group=group_name,
-                         help='Storage account name. Must be used in conjunction with either '
-                         'storage account key or a SAS token. Environment variable: '
-                         'AZURE_STORAGE_ACCOUNT')
+                         help='Storage account name. Must be used in conjunction with either storage account key or '
+                              'a SAS token. If both are missing, the command will try to query the storage account key'
+                              ' using the currect Azure account. If a large number of storage commands are executed '
+                              'the API quote may be hit. Environment variable: AZURE_STORAGE_ACCOUNT')
```
Comment (tjprescott) [inline src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_command_type.py]: typo: "API quote" > "API quota"

Comment (troydai) [reply src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_command_type.py]: Aha!

**Comment by troydai** [comment]
@derekbekoe  can you sign off again?

**Pull request merged**